src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -1415,6 +1436,21 @@ struct __ServiceCallContext {
 
 typedef struct __ServiceCallContext* PServiceCallContext;
 
+typedef struct __StreamEventMetadata StreamEventMetadata;
+struct __StreamEventMetadata {

```

Comment:
We should version this struct.  For other existing versioned structs what we do is if the current default version is higher than the one the client provides we fill in defaults for the fields that don't exist in older versions.  

Right now we have an s3 prefix field which breaks encapsulation but also is specific to an event type, future event types may also have other things specific to them that need to be specified in this struct.

Reply:
added versioning in last commit